waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/conf/WaggleDanceConfiguration.java

```diff
@@ -45,6 +45,7 @@
   private int thriftServerStopTimeoutValInSeconds = 60;
   private int thriftServerRequestTimeout = 20;
   private TimeUnit thriftServerRequestTimeoutUnit = TimeUnit.SECONDS;
+  private Long statusPollingDelay = null;
```

Comment:
Why is this modelled as a `Long` and not a `long` with a default like the other similar values above?

Reply:
default is in the annotation